src/renderer/hooks/customHooks.ts

```diff
@@ -0,0 +1,25 @@
+import { useEffect } from 'react';
+
+function useCustomHook(
```

Comment:
I reckon you name the custom hook `useKeypress` and then instead of manually/statically checking 'Enter' and 'NumpadEnter', pass in an array of strings corresponding to keyboard press codes. Then in function check the array parameter (maybe `keycodes`) to see if `event.code` is in it.
This will make the hook more reusable without a huge amount of extra code.

Also, not sure if necessary for `keypress` event but if you're feeling up to it. Have a look at [my PR](https://github.com/chloebrett/mlvet/pull/173) where I use a debouncer (basically just spaces out function calls based on user input to stop app crashing) to stop the UI lagging/crashing on continuous input (not sure if a `keypress` event is a once off but if it isn't then the user holding down the key could cause lag).

Reply:
I did not end up implementing `useDebounce`. Here is the commit with the changes: [2cfe8cd](https://github.com/chloebrett/mlvet/pull/175/commits/2cfe8cdf2d2f5cadb584f5f938d5cbaa3a43f3f3).

I can already see how the code can become more reusable. Nice one Hugh